gqlalchemy/query_builder.py

```diff
@@ -451,29 +451,32 @@ def from_(
 
         return self
 
-    def where(self, property: str, operator: str, value: Any) -> "DeclarativeBase":
+    def where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a WHERE statement Cypher partial query."""
         value_cypher = to_cypher_value(value)
+        separator = "" if operator == ":" else " "
```

Comment:
Extract string constants

Reply:
Extract where?
;Something like:
        self._query.append(WhereConditionPartialQuery(
            WhereConditionConstants.WHERE, 
            ("" if operator == ":" else " ").join([item, operator, value_cypher])
            ))
?
is it not more readable as is?

Reply:
No, we are hard-coding this operator ":"